Source/WebCore/page/Page.cpp

```diff
@@ -3794,6 +3815,33 @@ void Page::setFullscreenControlsHidden(bool hidden)
 #endif
 }
 
+Document* Page::outermostFullscreenDocument() const
```

Comment:
Not sure outermost is how I would describe it. If anything, it feels more inner than outer.    `deepest` described it better IMO.

Reply:
Happy to paint this bikeshed whatever color people agree upon.